vaadin-context-menu-flow-parent/vaadin-context-menu-flow/src/main/java/com/vaadin/flow/component/contextmenu/GeneratedVaadinContextMenu.java

```diff
@@ -352,4 +346,34 @@ protected Registration addOpenedChangeListener(
                                 new OpenedChangeEvent<R>((R) this,
                                         event.isUserOriginated())));
     }
+
+    /**
+     * Has no effect for ContextMenu because the element for ContextMenu is not
+     * clickable. Therefore, this method should not be used.
+     *
+     * @see #addClickListener(ComponentEventListener)
+     *
+     * @deprecated since 23.3
+     */
+    @Deprecated()
+    @Override
+    public Registration addClickListener(
+            ComponentEventListener<ClickEvent<R>> listener) {
+        return ClickNotifier.super.addClickListener(listener);
+    }
+
+    /**
+     * Has no effect for ContextMenu because the element for ContextMenu is not
+     * clickable. Therefore, this method should not be used.
+     *
+     * @see #addClickShortcut(Key, KeyModifier...)
+     *
+     * @deprecated since 23.3
+     */
+    @Deprecated()
```

Comment:
```suggestion
    @Deprecated
```
Nit: No need for the parenthesis